docs/faq.md

```diff
@@ -1,21 +1,179 @@
+
+* **Sustainability:** No not saving the planet (although compute efficiencies of serverless will certainly have that effect), I mean many projects are flashes in the pan. It’s easy to release something and get to the front of hacker news for a few days, but it’s much harder to sustain the momentum, community, and vision all while maintaining technical and usability excellence. Our team founded and built a successful startup so we know the difficulties of the journey and now at Oracle we’re excited to see the complete vision through. We’re ready to run the marathon it takes to build a great and lasting project.
+
+### What are the key components of Fn?
+
+The Fn Project today consists of 4 major components:
+
+1. Fn Server is the Functions-as-a-Service system that allows developers to easily build, deploy, and scale their functions into a multi-cloud environment. It’s fast, reliable, scalable, and container-native, which I’ll discuss more below.
+
+2. The Fn Load Balancer (Fn LB) allows operators to deploy clusters of Fn servers and route traffic to them intelligently. Most importantly, it will route traffic to nodes where hot functions are running to ensure optimal performance, as well as distribute load if traffic to a specific function increases. It also gathers information about the entire cluster which you can use to know when to scale out (add more Fn servers) or in (decrease Fn servers).
+
+3. Fn FDK’s — Starting with Java, we are releasing a number of FDK’s, or Function Development Kits, aimed at quickly bootstrapping functions in all languages, providing a data binding model for function inputs, make testing your functions easier, as well as lay the foundation for building more complex serverless applications.
+
+4. Fn Flow allows developers to build and orchestrate higher level workflows of functions all inside their programming language of choice. It makes it easy to use parallelism, sequencing/chaining, error handling, fan in/out, etc., without learning complicated external models built with long JSON or YAML templates. Best of all, Flow tracks all of the function call graphs, allowing for visualization in the dashboard, full stack logs of entire “Flows”, and variable/memory reconstitution throughout the entire function graph.
+
+### Why open source Fn?
+
+We believe that an open container native cloud platform based on Docker and Kubernetes is the future.  As such we want to ensure that anyone can write and deploy functions to any cloud provider so that customers have choice.  That said, we intend to compete hard to make sure that our cloud infrastructure is the best platform to run those functions.
+
+<a id="Technical"></a>
+## Technical
+
+### What languages can be used to write functions on Fn?
+
+Out of the box support includes: Java, Go, Ruby, Python, PHP, Rust, .NET Core, and Node.js (including AWS Lambda compatibility).
```

Comment:
probably we should say here which FDKs are officially supported

Reply:
So now I'm just listing the 4 initially supported languages. Java, Go, Python, Node.js.